lib/plugins/project-json/index.js

```diff
@@ -0,0 +1,23 @@
+import processJSON from '../helper/process-json';
+import { jsonRegExKeyValue } from '../helper/project-json-regex-builder';
+import insertLink from '../../insert-link';
+
+function linkDependency(blob, key, value) {
+  const regex = jsonRegExKeyValue(key, value);
+
+  insertLink(blob.el, regex, {
+    resolver: 'nugetUrl',
+    target: '$1',
+    type: 'project_json',
```

Comment:
According to your `nuget-url.js` resolver this property is not needed. 

It's used in the `live-resolver-query` to differentiate between the different registries so our https://github.com/OctoLinker/live-resolver server knows what to do.